packages/calling-stateful-client/src/StreamUtils.ts

```diff
@@ -511,7 +572,11 @@ export function createView(
     // Render LocalVideoStream that is not part of a Call
     return createViewUnparentedVideo(context, internalContext, stream, options);
   } else {
-    console.warn('Invalid combination of parameters');
+    _logEvent(callingStatefulLogger, {
+      name: EventNames.CREATE_STREAM_INVALID_PARAMS,
+      level: 'warning',
```

Comment:
I agree with this being a `warning`. Client used the API incorrectly, not an internal error.